receiver/awsxrayreceiver/internal/translator/sdk.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awsxrayreceiver/internal/tracesegment"
+)
+
+func populateInstrumentationLibrary(seg *tracesegment.Segment, ils *pdata.InstrumentationLibrarySpans) {
+	if seg.AWS != nil && seg.AWS.XRay != nil {
```

Comment:
This should be `Resource.sdk.name` / language / version I think. Unfortunately we'll need to parse into a name and language. The exporter ignores instrumentationlibrary so you can fill it with any dummy value

Reply:
Gonna add these information to both the `instrumentationlibrary` and as resource attributes